Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN \
     cd app && go build -o /build/doku -ldflags "-X main.revision=${version} -s -w"
 
 
-FROM node:16.14.2-alpine as build-frontend
+FROM node:16.20-alpine as build-frontend
```

Comment:
This version is quite new https://hub.docker.com/layers/library/node/16.20-alpine/images/sha256-72e89a86be58c922ed7b1475e5e6f151537676470695dd106521738b060e139d?context=explore